### PR TITLE
CCD-gateway-REFDATA-Configuration

### PR DESCRIPTION
**Pull request by jamesohmcts**
**Before creating a pull request make sure that:**

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-279

### Change description ###
Reference data is data that is used to structure and constrain other data. It is typically stable information with a known set of values that rarely change. As the name suggests, reference data is designed to be referenced by a wide variety of other data. This is done in order to create a standard vocabulary and structure across diverse systems and data sources.

Currently, we have a single reference data team to create a set of standardised API services against two classes of data:

Professional and user data – such as users and their preferences, Organisations, their membership and relevant account codes – and API’s to support create, update, read and delete operations against this data. This is operational data as far as CFT is concerned.

Reference data – such as courts and venues, judges and their tickets and similar – presented as ‘read only’ via API’s. We will not manage true reference data. We will just make it available via standard end points and via a standard methodology. This is needed generally across CFT, but particularly by the Task Allocation / workflow work currently being kicked off.

 Reference Data is back end APIs and consuming services like Expert UI going to access is these through public gateway using CCD API proxy, hence needed this change.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -115,6 +115,11 @@ applyProxy(app, {
   ]
 });
 
+applyProxy(app, {
+  source: '/refdata/**'
+  target: config.get('proxy.refdata')
```
Comment (mario-paniccia) [inline app.js]: has Reference Data anything to do with CCD? 
Why ExUI needs to access that backend through the CCD Gateway?

File: config/default.yaml [hunk 2]
```diff
@@ -6,6 +6,7 @@ proxy:
   print_service: https://return-case-doc.dev.ccd.reform.hmcts.net
   case_activity: http://localhost:3460
   payments: http://localhost:9999/payments
+  refdata: http://localhost:8090/refdata
```
Comment (mario-paniccia) [inline config/default.yaml]: where is configuration for all the other environments?

File: app.js (hunk 1 above)
Comment (jamesohmcts) [reply app.js]: It is tactical solution until ref data has it own API gateway.

File: config/default.yaml (hunk 2 above)
Comment (jamesohmcts) [reply config/default.yaml]: Can you point to the configuration files for the other environments?

Comment (mario-paniccia) [reply config/default.yaml]: please have  a look at aat.tfvars, variables.tf, main.tf under folder infrstructure

**Pull request merged**